source/frontend/src/features/mapSideBar/lease/tabs/consultations/detail/ConsultationListView.tsx

```diff
@@ -0,0 +1,222 @@
+import { useMemo } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaPlus, FaTrash } from 'react-icons/fa';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { StyledRemoveLinkButton } from '@/components/common/buttons/RemoveButton';
+import ContactFieldContainer from '@/components/common/ContactFieldContainer';
+import EditButton from '@/components/common/EditButton';
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { StyledSummarySection } from '@/components/common/Section/SectionStyles';
+import { SectionListHeader } from '@/components/common/SectionListHeader';
+import * as API from '@/constants/API';
+import Claims from '@/constants/claims';
+import useKeycloakWrapper from '@/hooks/useKeycloakWrapper';
+import useLookupCodeHelpers from '@/hooks/useLookupCodeHelpers';
+import { getDeleteModalProps, useModalContext } from '@/hooks/useModalContext';
+import { ApiGen_Concepts_ConsultationLease } from '@/models/api/generated/ApiGen_Concepts_ConsultationLease';
+import { prettyFormatDate } from '@/utils';
+import { booleanToYesNoUnknownString } from '@/utils/formUtils';
+
+export interface IConsultationListViewProps {
+  loading: boolean;
+  consultations: ApiGen_Concepts_ConsultationLease[];
+  onAdd: () => void;
+  onEdit: (consultationId: number) => void;
+  onDelete: (consultationId: number) => void;
+}
+
+interface GroupedConsultations {
+  consultationTypeCode: string;
+  consultationTypeDescription: string;
+  consultations: ApiGen_Concepts_ConsultationLease[];
+  hasItems: boolean;
+}
+
+export const ConsultationListView: React.FunctionComponent<IConsultationListViewProps> = ({
+  loading,
+  consultations,
+  onAdd,
+  onEdit,
+  onDelete,
+}) => {
+  const keycloak = useKeycloakWrapper();
+  const history = useHistory();
```

Comment:
See github actions warnings about unused variables.

Reply:
I think you have seen an old commit?